action.yml

```diff
@@ -171,13 +171,14 @@ runs:
           jq < response.json
         fi
 
-        # Store REQ_ID into GITHUB_ENV variables
         req_id=$(jq -r .id response.json)
-        tf_url=$(jq -r .run.artifacts response.json)
-        echo "REQ_ID=$req_id" >> $GITHUB_ENV
+        # NOTE(ivasilev) Do not store REQ_ID and RESULTS_URL as GITHUB_ENV variables, as in case multiple
+        # testing farm requests are scheduled subsequent VAR=VAL >> $GITHUB_ENV instructions are ignored and
+        # information from first run only is used.
         # NOTE(ivasilev) Target url may not be available at this moment if the job is still queued, so
         # will rather construct one from artifact_url and id as it has predictable format
-        echo "RESULTS_URL=${{ inputs.artifacts_url }}/$req_id" >> $GITHUB_ENV
+        echo "::set-output name=req_id::$req_id"
+        echo "::set-output name=results_url::${{ inputs.artifacts_url }}/$req_id"
```

Comment:
The changes are fine but what will be URL in the case of public ranch and in the case of private ranch?
I mean, artifacts.dev.testing-farm.io for public ranch and osci.... for private ranch?

Are you sure with this change?

Reply:
As far as I know, artifact_url can be taken from $(jq -r .run.artifacts response.json)?

See our working GitHub Action tests, like here: https://github.com/sclorg/s2i-nodejs-container/pull/318

Reply:
@phracek We can't rely on artifact_url being present there all the time as it will be there only when the request is already scheduled (has new or running state), but not when it is still waiting to be scheduled.
@thrix Please correct me here if I am wrong

Reply:
As for the url composition - I didn't change the logic here, all I did was refactor 
```
echo "RESULTS_URL=${{ inputs.artifacts_url }}/$req_id" >> $GITHUB_ENV
```
into 
```
echo "::set-output name=results_url::${{ inputs.artifacts_url }}/$req_id"
```